src/main/java/frc/robot/commands/BangBangDriveCommand.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.DriveConstants;
+import frc.robot.subsystems.Drivetrain;
+import java.awt.geom.Point2D;
+
+public class BangBangDriveCommand extends Command{
+    Drivetrain driv;
```

Comment:
Make sure to use private when creating variables like driv, it automatically saves as package-private if you don't specify, but you want restricted access to the class where it's declared